client/my-sites/post-share/index.jsx

```diff
@@ -169,7 +205,7 @@ class PostShare extends Component {
 			onClick={ this.sharePost }
 			disabled={ this.isButtonDisabled() }
 		>
-			{ translate( 'Share post' ) }
+			{ this.state.scheduledDate ? translate( 'Schedule post' ) : translate( 'Share post' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Scheduled Post' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/publicize/schedule-button)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).